ftw/publisher/monitor/adapters.py

```diff
@@ -29,14 +30,24 @@ def send_email(self):
         """Sends the email
         """
 
-        properties = getUtility(IPropertiesTool)
         mh = getToolByName(self.context, 'MailHost')
-        from_addr = properties.email_from_address
+
+        if IS_PLONE_5:
+            from Products.CMFPlone.interfaces.controlpanel import IMailSchema
+            from plone.registry.interfaces import IRegistry
+            registry = getUtility(IRegistry)
+            mail_settings = registry.forInterface(IMailSchema, prefix='plone')
+            from_name = mail_settings.email_from_name
+            from_addr = mail_settings.email_from_address.decode('ascii')
```

Comment:
Why do we decode it as ASCII in Plone 5 whereas we decode it to utf-8 on Plone 4?
I saw in the comment "The email address is an ASCII field", is that defined by Plone or is it just "emails should be ascii"?

Reply:
> Why do we decode it as ASCII in Plone 5 whereas we decode it to utf-8 on Plone 4?
> I saw in the comment "The email address is an ASCII field", is that defined by Plone or is it just "emails should be ascii"?

I need a `ISO-88...` encoded string later. The `decode ascii` is just there to be explicit, python would chose anyway this  encoding. once we encode to something else. And yes it's also explicitly a TextAscii field  in the schema.